GAE/src/org/waterforpeople/mapping/app/gwt/client/survey/QuestionDto.java

```diff
@@ -26,458 +26,466 @@
-    }
-
-    public void setImmutable(Boolean immutable) {
-        this.immutable = immutable;
-    }
-
-    public Boolean getImmutable() {
-        return immutable;
-    }
-
-    public List<Long> getQuestionOptions() {
-        return questionOptions;
-    }
-
-    public void setQuestionOptions(List<Long> questionOption_ids) {
-        this.questionOptions = questionOption_ids;
-    }
-
-    public Long getSourceId() {
-        return sourceId;
-    }
-
-    public void setSourceId(Long sourceId) {
-        this.sourceId = sourceId;
-    }
-
-    public Boolean getLocaleNameFlag() {
-        return localeNameFlag;
-    }
-
-    public Boolean getGeoLocked() {
-        return geoLocked;
-    }
-
-    public void setGeoLocked(Boolean geoLocked) {
-        this.geoLocked = geoLocked;
-    }
-
-    public void setLocaleNameFlag(Boolean localeNameFlag) {
-        this.localeNameFlag = localeNameFlag;
-    }
-
-    public Boolean getAllowExternalSources() {
-        return allowExternalSources;
-    }
-
-    public void setAllowExternalSources(Boolean allowExternalSources) {
-        this.allowExternalSources = allowExternalSources;
-    }
-
-    public Boolean getLocaleLocationFlag() {
-        return localeLocationFlag;
-    }
-
-    public void setLocaleLocationFlag(Boolean localeLocationFlag) {
-        this.localeLocationFlag = localeLocationFlag;
-    }
-
-    public Boolean getRequireDoubleEntry() {
-        return requireDoubleEntry;
-    }
-
-    public void setRequireDoubleEntry(Boolean requireDoubleEntry) {
-        this.requireDoubleEntry = requireDoubleEntry;
-    }
-
-    public String getQuestionId() {
-        return questionId;
-    }
-
-    public void setQuestionId(String questionId) {
-        // Missing questionId is represented as null
-        if (questionId != null && questionId.matches("\\s*")) {
-            this.questionId = null;
-        } else {
-            this.questionId = questionId;
-        }
-    }
-
-    public Long getCascadeResourceId() {
-        return cascadeResourceId;
-    }
-
-    public void setCascadeResourceId(Long cascadeResourceId) {
-        this.cascadeResourceId = cascadeResourceId;
-    }
-
-    public List<String> getLevelNames() {
-        return levelNames;
-    }
-
-    public void setLevelNames(List<String> levelNames) {
-        this.levelNames = levelNames;
-    }
-
-    public void setAllowPoints(Boolean allowPoints) {
-        this.allowPoints = allowPoints;
-    }
-
-    public Boolean getAllowPoints() {
-        return allowPoints;
-    }
-
-    public void setAllowLine(Boolean allowLine) {
-        this.allowLine = allowLine;
-    }
-
-    public Boolean getAllowLine() {
-        return allowLine;
-    }
-
-    public void setAllowPolygon(Boolean allowPolygon) {
-        this.allowPolygon = allowPolygon;
-    }
-
-    public Boolean getAllowPolygon() {
-        return allowPolygon;
-    }
+	private static final long serialVersionUID = -4708385830894435407L;
```

Comment:
Unrelated changes broke indentation. You can always review your diff before committing and pushing